symphony-bdk-examples/bdk-app-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -0,0 +1,18 @@
+
+bdk:
+  host: devx1.symphony.com
+  bot:
+    username: fxbot-hong
+    privateKeyPath: ${user.home}/.symphony/keys/rsa-private-bot.pem
+
+  app:
+    auth.enabled: true
+    auth.jwt-cookie:
+      enabled: true
+      expire-in: 43200
+    appId: testapp-hongle
+    privateKeyPath: ${user.home}/.symphony/keys/app_privatekey.pem
```

Comment:
```yaml
app:
  appId: testapp-hongle
  privateKeyPath: ${user.home}/.symphony/keys/app_privatekey.pem
  auth:
    enabled: true # activates AppAuthController 
    jwt-cookie:
      enabled: true
      max-age: 43200 # in seconds
```

Reply:
I would use `jwtCookie` and `maxAge` instead to make the format consistent

Reply:
You could check, but if I remember well both are supported by Spring yaml parser